torch/export/__init__.py

```diff
@@ -50,7 +50,7 @@
     "ExportedProgram",
     "ModuleCallEntry",
     "ModuleCallSignature",
-    "core_aten_decompositions",
```

Comment:
oh I didn't realize `torch.export.core_aten_decompositions` was a public API. are we ok with fully deleting it in this PR?

Reply:
Yeah I added this a couple weeks ago, this diff just renames it.